Leafly-com/lm-inventory-service
# Lmdisc 102 skubana inventory sync handler

# Description
Adds `sync-inventory-stocks` npm script that retrieves inventory from Skubana and bulk upserts it to the Inventory collection

## Manual Testing
Update `lm-inventory-service/src/config/index.ts` `skubana/accessToken` and `skubana.environment` (if necessary) to a valid configuration. (Dev Access token TBD)

`docker-compose up -d` (if necessary)
`npm run seed:local` (if data already not in db)
`npm run sync-inventory-stocks`

Check the Mongo db and confirm that all Product Stocks in the skubana account are inserted into the Mongo DB. Any existing skus should have their `availableQuantity` updated.

![image](https://user-images.githubusercontent.com/54331819/66434472-4329ad00-e9e8-11e9-9181-912d9bc99db7.png)

The response in the log should have the results of the operation:
![image](https://user-images.githubusercontent.com/54331819/66594574-0633f680-eb5f-11e9-9715-cba0fa5d398e.png)


## Unit Testing
`npm run test`
![image](https://user-images.githubusercontent.com/54331819/66594650-25cb1f00-eb5f-11e9-9ce9-c39e93ab2c55.png)
